python/GafferUITest/MenuBarTest.py

```diff
@@ -236,6 +236,67 @@ def buildMenu( identifier ) :
 
 		self.assertEqual( callCounts, { "MenuA" : 1, "MenuB" : 0 } )
 
+	def testAddShortcutTarget( self ) :
```

Comment:
Thanks! I should have added some test coverage for this in the first place.

Reply:
No worries! I felt much better about leaning on it in such a conspicuous location with a test in place...